onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -231,12 +291,14 @@ def optimize_onnx(
         auto_mixed_precision: bool = True,
         use_gpu: bool = False,
         provider: str = "cpu",
+        model_impl: str = "hf",
     ):
         """Optimize ONNX model with an option to convert it to use mixed precision."""
 
         from fusion_options import FusionOptions
 
         optimization_options = FusionOptions("bart")
+        optimization_options.model_impl = model_impl
```

Comment:
Same comment that I don't think we need this once the fusion changes are in `fusion_bart_attention.py`.